assets/install.sh

```diff
@@ -7,47 +7,54 @@
 ##
 
 os="$(uname -s)"
-packages="ruby ruby-dev"
+packages="ruby ruby-dev git"
 
-# install_file TYPE FILENAME
-# TYPE is "rpm", "deb", "solaris", "sh", etc.
-install_file() {
-  echo "Installing the ruby things"
+# install_deps PLATFORM
+# PLATFORM is "redhat", "debian", "freebsd", "arch", etc.
+install_deps() {
+  echo "Installing dependencies"
   case "$1" in
     "redhat")
       echo "installing with yum..."
-      packages="ruby ruby-devel"
+      packages="ruby ruby-devel git"
+      # shellcheck disable=SC2086
```

Comment:
If the above is changed, then you don't need this, and instead you can do 

```
yum install -y "${packages[@]}"
```

Which would be much better

Reply:
Why are we trying to go for arrays when this seems like a simple change?